Tests/Resources/app/AppKernel.php

```diff
@@ -28,9 +28,10 @@ public function registerBundles()
      */
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $this->isDebug()
-            ? $loader->load(__DIR__.'/config/config_debug.yml')
-            : $loader->load(__DIR__.'/config/config.yml');
+        $loader->load(__DIR__.'/config/config_'.$this->getEnvironment().'.yml');
+        if ($this->isDebug()) {
+            $loader->load(__DIR__.'/config/config_debug.yml');
+        }
```

Comment:
`config.yml` have been renamed to `config_test.yml` to always have a configuration file matching the current environment.